src/vellum/workflows/nodes/displayable/final_output_node/node.py

```diff
@@ -27,6 +28,9 @@ def __new__(mcs, name: str, bases: Tuple[Type, ...], dct: Dict[str, Any]) -> Any
             **annotations,
             "value": parent.get_output_type(),
         }
+
+        mcs._validate_output_type_consistency(parent)
```

Comment:
[blocking] I thought we landed on these methods being called during serialize instead of on import? On import prevents workflows from even running at all, even if its unused